anvio/docs/artifacts/misc-data-item-orders.md

```diff
@@ -0,0 +1,5 @@
+This file describes the **order of branches for a visualization**.
+
+Most often, this is a phylogenetic tree, but it can also just describe what order to display the branches or items in. 
+
+As of now, this is an output from programs that generate a tree of this kind, but is not used by any anvi'o programs. 
```

Comment:
>but is not used by any anvi'o programs.

I am not super-duper familiar with interactive, but I thought that there was a way to import item orders into anvi'o for the interactive interface (I think I saw it here: http://merenlab.org/2017/12/11/additional-data-tables/). Or maybe I am mixing it up with layer orders 😕

Reply:
I haven't looked through that article yet (I'll be sure to do so), but it was my impression that anvi-import-misc-data worked on layer-orders. It might also work with item-orders though, I'm not sure.  :/